charts/scalardb/values.yaml

```diff
@@ -7,6 +7,98 @@ nameOverride: ""
 # fullnameOverride -- String to fully override scalardb.fullname template
 fullnameOverride: ""
 
+envoy:
+  # envoy.replicaCount -- number of replicas to deploy
+  replicaCount: 3
+
+  envoyConfiguration:
+    # envoy.envoyConfiguration.adminAccessLogPath -- admin log path
+    adminAccessLogPath: /dev/stdout
+
+  image:
+    # envoy.image.repository -- Docker image
+    repository: ghcr.io/scalar-labs/scalar-envoy
+    # envoy.image.tag -- Docker tag
+    version: 1.1.0
```

Comment:
Need to update new envoy image with https://github.com/scalar-labs/scalar-envoy/pull/13

Reply:
As I commented, the new update in envoy is nice, but it has a backward compatibility issue.
I think we should map ports on the k8s side.

Reply:
OK! I fixed the port of `scalardb` container to `50051` in 5af1f2c.